Cubical/Categories/Displayed/Constructions/HomPropertyOver.agda

```diff
@@ -0,0 +1,113 @@
+module Cubical.Categories.Displayed.Constructions.HomPropertyOver where
+
+open import Cubical.Foundations.Prelude
+
+open import Cubical.Data.Unit
+
+open import Cubical.Categories.Category
+open import Cubical.Categories.Displayed.Base
+open import Cubical.Categories.Displayed.Constructions.StructureOver
+
+private
+  variable
+    ℓC ℓC' ℓP : Level
+
+open Category
+
+module _
+  (C : Category ℓC ℓC') (P : ∀ {x y} → C [ x , y ] → Type ℓP)
+  (Pprop : ∀ {x y} (f : C [ x , y ]) → isProp (P f))
+  (Pid : ∀ {x} → P (id C {x}))
+  (Pcomp : ∀ {x y z} (f : C [ x , y ]) (g : C [ y , z ]) → P f → P g → P (_⋆_ C f g))
+  where
```

Comment:
I think there's so many fields here that we should make this a struct `HomPropertyOver` like we do with `StructureOver`.

Reply:
I've changed `HomPropertyOver` to a record and created a defined to displayed categories.

Reply:
I've updated these names to match, except for `Hom[_]`. It caused a parser ambiguity between `Hom[_,_]` and `Hom[ (_,_) ]`. I used `Hom[_][-,-]` instead, which is the most reasonable idea I've had.